### PR TITLE
Updates the POD details output in workload get command

### LINKED ISSUE #224: Workload get output pod status should mimic kubectl output
### Description of problem

`tanzu apps workload get` shows the status of `Pending` when a build is running. The status `Pending` while accurate, is not very helpful. 

Screen Shot 2022-07-17 at 11 02 38 PM

### Proposed solution
`apps` plugin should mimic the output of `kubectl` as shown in the screenshot below. It should show `init` container information when the pods are not in a `completed` or `succeeded` status.

Screen Shot 2022-07-17 at 11 04 55 PM

cc @danfein for design recommendations.

### PR DESCRIPTION
**Pull request by shaheerkootteeri**
Pull request

### What this PR does / why we need it

PR will display the POD details same as that of `kubectl get pod  ` output command results. 

### Which issue(s) this PR fixes

Fixes #224

### Describe testing done for PR

Unit test cases added, also updated existing test cases to reflect the POD details output. 
Tested in local cluster.

### CODE REVIEW

File: pkg/cli-runtime/testing/client.go [hunk 1]
```diff
@@ -31,26 +41,318 @@ func (c *fakeclient) DefaultNamespace() string {
...
+			},
+			VersionedResources: map[string][]metav1.APIResource{
+				"v1beta1": {
+					{Name: "deployments", Namespaced: true, Kind: "Deployment"},
+					{Name: "replicasets", Namespaced: true, Kind: "ReplicaSet"},
+				},
+			},
+		},
+		{
+			Group: metav1.APIGroup{
```
Comment (shashwathi) [inline pkg/cli-runtime/testing/client.go:215]: Keep this group and delete the rest for now

File: pkg/cli-runtime/testing/command_table.go [hunk 2]
```diff
@@ -82,6 +91,9 @@ type CommandTestCase struct {
 	// GivenObjects represents resources that would already exist within Kubernetes. These
 	// resources are passed directly to the fake client.
 	GivenObjects []client.Object
+	// UnstructuredObjects represents resources that would already exist within Kubernetes. These
+	// resources are passed directly to the fake client.
+	UnstructuredObjects []client.Object
```
Comment (shashwathi) [inline pkg/cli-runtime/testing/command_table.go:96]: you do not need this anymore. You are checking givenObject kind to decide whether to use builder

File: pkg/cli-runtime/testing/client.go (hunk 1 above)
Comment (shaheerkootteeri) [reply pkg/cli-runtime/testing/client.go]: I have updated the code to remove other types.

File: pkg/cli-runtime/testing/command_table.go (hunk 2 above)
Comment (shaheerkootteeri) [reply pkg/cli-runtime/testing/command_table.go]: I have removed the `unstructuredObjects`

**Comment by shaheerkootteeri** [comment]
Closing the pull request and will reinitiate from

**Pull request closed without merging**